src/migrations/20220706152116-add-uei-to-recipient.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => queryInterface.addColumn(
+    'Recipients',
+    'uei',
+    { type: Sequelize.DataTypes.STRING, defaultValue: '', allowNull: true },
+    { transaction },
+  )),
```

Comment:
```suggestion
  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
  try {
        const loggedUser = '0';
        // const transactionId = '';
        const sessionSig = __filename;
        const auditDescriptor = 'RUN MIGRATIONS';
        await queryInterface.sequelize.query(
          `SELECT
            set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
            set_config('audit.transactionId', NULL, TRUE) as "transactionId",
            set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
            set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
          { transaction },
        );
      } catch (err) {
        console.error(err); // eslint-disable-line no-console
        throw (err);
      }
      
  await queryInterface.addColumn(
    'Recipients',
    'uei',
    { type: Sequelize.DataTypes.STRING, defaultValue: '', allowNull: true },
    { transaction },
  );
  }),
```

Reply:
Also, this type of change will cause a record to be added to the audit log for every row in the table, if we dont want that, the audit logging should be suspended.